### PR TITLE
Standardized article lighthouse a11y test

### LINKED ISSUE #4065: Use standardised test article for Lighthouse/a11y tests
**Is your feature request related to a problem? Please describe.**
As part of https://github.com/bbc/simorgh/issues/3146 we started running our e2e tests against the new standardised test article for News. This article includes a piece of video content.

We should make sure that our automated Lighthouse and a11y tests also check this article to give representative results and catch bugs and regressions related to AV.

**Describe the solution you'd like**
- Review existing assets tested by a11y tests and Lighthouse
- Add to (or edit existing) assets
- Run a11y and Lighthouse tests and fix problems found
   - Liaise with core pod about best way to validate behaviour of tests on test this before merge
- Agree plan for scaling this to other services' standardised articles

**Describe alternatives you've considered**
None.

**Testing notes**

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves #4065 

**Overall change:** Use standardized article for lighthouse/a11y tests

**Code changes:**

- Added lighthouse command to run for standardized article
- Changed article id on a11y  script

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "main": "./src/client.js",
   "scripts": {
     "amp:validate": "wait-on http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/igbo.amp && amphtml-validator http://localhost.bbc.com:7080/korean/bbc_korean_radio/liveradio.amp",
-    "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && bbc-a11y",
-    "bbcA11y:ci": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && xvfb-run -a bbc-a11y",
+    "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/cn7k01xp8kxo && bbc-a11y",
+    "bbcA11y:ci": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/cn7k01xp8kxo && xvfb-run -a bbc-a11y",
```
Comment (jamesdonoh) [inline package.json:9]: How does `bbc-a11y` know which asset to test?

File: scripts/lighthouseRun.sh [hunk 2]
```diff
@@ -2,5 +2,6 @@
 set -e
 
 npx -p lighthouse@5 lighthouse http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
+npx -p lighthouse@5 lighthouse http://localhost.bbc.com:7080/news/articles/cn7k01xp8kxo --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
```
Comment (jamesdonoh) [inline scripts/lighthouseRun.sh:4]: Is there a reason to retain `...8po` here? I think we should be replacing it.

**Review by jamesdonoh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rhenshaw56) [reply package.json:9]: @jamesdonoh [it uses](https://github.com/bbc/simorgh/blob/latest/a11y.js#L4) the path set in [cypress services config](https://github.com/bbc/simorgh/blob/latest/cypress/support/config/services.js#L776)

Comment (jamesdonoh) [reply package.json:9]: Ah okay, so the article ID we specify on these two lines is irrelevant then. But we might as well change it for consistency.

**Review by jamesdonoh — APPROVED** [review]
Looks good.

**Review by dr3 — APPROVED** [review]

**Pull request merged**